src/main/java/com/halotroop/arrchecker/ARRChecker.java

```diff
@@ -16,15 +17,23 @@
 
     @Override
     public void onInitialize() {
+        final List<String> ignore = ImmutableList.of(
+                "minecraft",
+                "fabric-language-scala",
+                "com_moandjiezana_toml_toml4j",
+                "java");
+        final List<String> fuzzyIgnore = ImmutableList.of(
+                "org_codehaus_groovy_groovy",
+                "org_jetbrains_kotlin");
         arrMods.clear();
         for (ModContainer mod : FabricLoader.getInstance().getAllMods()) {
             ModMetadata modMeta = mod.getMetadata();
             // Assume all authors don't know how capitalization works, and don't use an array for the string!
-            String modLicense = modMeta.getLicense().toString().toLowerCase().replace('[', ' ').replace(']', ' ').trim();
+            String modLicense = modMeta.getLicense().toString().toLowerCase().replace('[', ' ').replace(']', ' ').replace('-', ' ').trim();
```

Comment:
This is getting verbose, and I realized earlier that by doing this we're adding extra spaces.

Replace this line with a regex for `[`, `]`, and `-`. It should replace with an empty string rather than a space.